package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "start": "node app.js",
-    "lint": "eslint --ext .js,.json .",
+    "lint": "eslint --ext .js,.json . && cat .bithoundrc | eslint --stdin --stdin-filename=f.json",
```

Comment:
Discussed this and agreed it wasn't important. If it does become an issue we can solve at that time.